website/content/partials/known-issues/transit-managed-keys-sign-fails.mdx

```diff
@@ -0,0 +1,16 @@
+### Transit Sign API calls with managed keys fail
+
+#### Affected versions
+
+- 1.14.0+ up to 1.14.3 inclusively.
+
+#### Issue
+
+The Transit sign API call responds with an error "requested version for signing does not contain a private part" when
+a Transit sign request uses a managed key.
```

Comment:
```suggestion
Vault responds to Transit sign API calls with the following error when the request uses a managed key:

`requested version for signing does not contain a private part`
```

Style correction: Make key information easy to scan for, make it clear who/what is taking action

Reply:
Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b and updated missing newline within https://github.com/hashicorp/vault/pull/23080/commits/8672872bc96e136c707c01da8d32cc08e0f97c2f